stdlib/2.7/codecs.pyi

```diff
@@ -68,7 +69,7 @@ def register(search_function: Callable[[str], CodecInfo]) -> None:
 def open(filename: str, mode: str = ..., encoding: str = ..., errors: str = ..., buffering: int = ...) -> StreamReaderWriter:
     ...
 
-def EncodedFile(file: BinaryIO, data_encoding: str, file_encoding: str = ..., errors = ...) -> 'StreamRecoder':
+def EncodedFile(file: BinaryIO, data_encoding: str, file_encoding: str = ..., errors: Any = ...) -> 'StreamRecoder':
```

Comment:
`Any` should already be the default for parameters without a type annotation. So there shouldn't be any need to change this line.

(There many other changes like this in this pull request)